src/components/demo/DOITooltipDemo.tsx

```diff
@@ -37,20 +37,68 @@ const DOITooltipDemo = (props: Props) => {
         }
     }
 
+    /* Function to map Handle or DOI response to valid record */
+    const FormatResponse = (responseRecord: {[name: string]: any}) => {
+        const record = {
+            data: {
+                attributes: {
+                    referentName: responseRecord.values[16].data.value,
+                    specimenHost: responseRecord.values[18].data.value,
+                    specimenHostName: responseRecord.values[19].data.value,
+                    primarySpecimenObjectId: responseRecord.values[20].data.value,
+                    topicDiscipline: responseRecord.values[27].data.value,
+                    livingOrPreserved: responseRecord.values[29].data.value
+                }
+            }
+        };
+
```

Comment:
remove extra space?

Reply:
I like to have space (enter) in between a return statement and what is happening above if its on the same indent level